--- origsrc/kdesvn-1.0.2/src/kdesvnd/CMakeLists.txt	2007-06-10 16:23:29.000000000 -0500
+++ src/kdesvn-1.0.2/src/kdesvnd/CMakeLists.txt	2008-09-24 05:18:21.843062800 -0500
@@ -13,10 +13,10 @@
 KDE3_ADD_KPART(kded_kdesvnd ${kded_kdesvnd_src})
 
 TARGET_LINK_LIBRARIES(kded_kdesvnd
-    svnqt
     ksvnwidgets
     kdesvnhelpers
     kdesvncfgreader
+    svnqt
     ${QT_AND_KDECORE_LIBS} ${KDE3_KIO_LIBRARY} ${KDE3_DCOP_LIBRARY}
     ${KDE3_UI_LIBRARY} ${KDE3_WALLET_LIBRARY} 
     ${KDE3_KINIT_KDED_LIBRARY})
