--- origsrc/kdesvn-1.3.2/src/kdesvnd/CMakeLists.txt	2009-02-25 06:11:30.000000000 -0600
+++ src/kdesvn-1.3.2/src/kdesvnd/CMakeLists.txt	2009-07-08 16:50:48.799361300 -0500
@@ -14,10 +14,10 @@ KDE4_ADD_PLUGIN(kded_kdesvnd ${kded_kdes
 KDE_ADD_MOC_FILES(kded_kdesvnd_src kdesvnd_dcop.h)
 
 TARGET_LINK_LIBRARIES(kded_kdesvnd
-    ${svnqt-name}
     ksvnwidgets
     kdesvnhelpers
     kdesvncfgreader
+    ${svnqt-name}
     ${KDE4_KIO_LIBS}
 )
 
