--- origsrc/kdesvn-1.6.0/src/helpers/CMakeLists.txt	2009-11-25 08:35:25.000000000 -0600
+++ src/kdesvn-1.6.0/src/helpers/CMakeLists.txt	2013-11-14 21:08:54.778621600 -0600
@@ -31,4 +31,4 @@ ADD_LIBRARY(kdesvnhelpers STATIC ${helpe
 
 SET_TARGET_PROPERTIES(kdesvnhelpers
     PROPERTIES
-    COMPILE_FLAGS ${CMAKE_SHARED_LIBRARY_CXX_FLAGS})
+    COMPILE_FLAGS "${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
--- origsrc/kdesvn-1.6.0/src/ksvnwidgets/CMakeLists.txt	2010-09-15 08:44:20.000000000 -0500
+++ src/kdesvn-1.6.0/src/ksvnwidgets/CMakeLists.txt	2013-11-14 21:09:09.674473600 -0600
@@ -75,6 +75,6 @@ KDE_AUTOMOC(${ksvnwidgetsrc})
 ADD_LIBRARY(ksvnwidgets STATIC ${kdesvn_shared_ui} ${ksvnwidgetsrc} ${ksvnwidgethdr})
 SET_TARGET_PROPERTIES(ksvnwidgets
     PROPERTIES
-    COMPILE_FLAGS ${CMAKE_SHARED_LIBRARY_CXX_FLAGS})
+    COMPILE_FLAGS "${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
 
 ADD_DEPENDENCIES(ksvnwidgets kdesvncfgreader)
--- origsrc/kdesvn-1.6.0/src/settings/CMakeLists.txt	2009-11-30 05:25:57.000000000 -0600
+++ src/kdesvn-1.6.0/src/settings/CMakeLists.txt	2013-11-14 21:09:17.604927200 -0600
@@ -53,7 +53,7 @@ ADD_DEPENDENCIES(settingsdlgs kdesvncfgr
 
 SET_TARGET_PROPERTIES(kdesvncfgreader settingsdlgs
     PROPERTIES
-    COMPILE_FLAGS ${CMAKE_SHARED_LIBRARY_CXX_FLAGS})
+    COMPILE_FLAGS "${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
 
 INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/kdesvn_part.kcfg DESTINATION ${KCFG_INSTALL_DIR})
 
--- origsrc/kdesvn-1.6.0/src/svnfrontend/CMakeLists.txt	2012-06-11 08:12:37.000000000 -0500
+++ src/kdesvn-1.6.0/src/svnfrontend/CMakeLists.txt	2013-11-14 21:09:24.464319500 -0600
@@ -138,6 +138,6 @@ ADD_SUBDIRECTORY(background)
 ADD_LIBRARY(svnfrontend STATIC ${svnfrontendsrc} ${frontend_ui_sources} ${hdr})
 SET_TARGET_PROPERTIES(svnfrontend
     PROPERTIES
-    COMPILE_FLAGS ${CMAKE_SHARED_LIBRARY_CXX_FLAGS})
+    COMPILE_FLAGS "${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
 
 ADD_DEPENDENCIES(svnfrontend ksvnwidgets svncharts)
--- origsrc/kdesvn-1.6.0/src/svnfrontend/background/CMakeLists.txt	2010-01-27 15:30:47.000000000 -0600
+++ src/kdesvn-1.6.0/src/svnfrontend/background/CMakeLists.txt	2013-11-14 21:09:54.686048100 -0600
@@ -45,6 +45,6 @@ ADD_LIBRARY(svnbackground STATIC
 
 SET_TARGET_PROPERTIES(svnbackground
     PROPERTIES
-    COMPILE_FLAGS ${CMAKE_SHARED_LIBRARY_CXX_FLAGS})
+    COMPILE_FLAGS "${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
 
 ADD_DEPENDENCIES(svnbackground ksvnwidgets)
--- origsrc/kdesvn-1.6.0/src/svnqt/CMakeLists.txt	2012-06-18 02:15:00.000000000 -0500
+++ src/kdesvn-1.6.0/src/svnqt/CMakeLists.txt	2013-11-14 21:09:30.873686100 -0600
@@ -189,7 +189,7 @@ ELSE(WIN32)
     SET(LIBRARY_OUTPUT_PATH ${PROJECT_BINARY_DIR}/lib${LIB_SUFFIX})
     SET_TARGET_PROPERTIES(${svnqt-name}
         PROPERTIES
-        COMPILE_FLAGS ${CMAKE_SHARED_LIBRARY_CXX_FLAGS})
+        COMPILE_FLAGS "${CMAKE_SHARED_LIBRARY_CXX_FLAGS}")
 ENDIF(WIN32)
 
 SET(LIB_MAJOR 7)
